app/grandchallenge/evaluation/views/api.py

```diff
@@ -23,3 +45,88 @@ class EvaluationViewSet(ReadOnlyModelViewSet):
         *api_settings.DEFAULT_RENDERER_CLASSES,
         PaginatedCSVRenderer,
     )
+
+    @action(detail=True, permission_classes=[CanClaimEvaluation])
```

Comment:
What methods does this use? Maybe better to be explicit about it here. It should not be GET as GET requests should not alter the objects. It should be a POST or PATCH request.

Reply:
Ah good point, it should be patch then IMO.